src/test/java/core/basesyntax/service/RegistrationServiceImplTest.java

```diff
@@ -0,0 +1,177 @@
+package core.basesyntax.service;
+
+import static org.junit.jupiter.api.Assertions.assertThrows;
+import static org.junit.jupiter.api.Assertions.assertTrue;
+
+import core.basesyntax.db.Storage;
+import core.basesyntax.exception.RegistrationUserException;
+import core.basesyntax.model.User;
+import org.junit.jupiter.api.AfterEach;
+import org.junit.jupiter.api.BeforeEach;
+import org.junit.jupiter.api.Test;
+
+class RegistrationServiceImplTest {
+    public static final String STRING_3_CHARACTER = "abc";
+    public static final String STRING_5_CHARACTER = "Ellis";
+    public static final String STRING_6_CHARACTER = "Morris";
+    public static final String STRING_7_CHARACTER = "Simpson";
+    public static final String EMPTY_STRING = "";
+    public static final int NEGATIVE_USER_AGE = -21;
+    public static final int USER_ZERO_AGE = 0;
+    public static final int USER_YOUNG_AGE = 17;
+    public static final int USER_ADULT_AGE = 18;
+    public static final int USER_USA_ADULT_AGE = 21;
```

Comment:
maybe make them more informative? If I see problem with USER_USA_ADULT_AGE, it's harder to understand, what's the difference between it and USER_ADULT_AGE. Put age in name